### PR TITLE
#415 Missing body for POST request in middleware

### PR DESCRIPTION
**Pull request by JHotterbeekx**
As described in issue 415, the body of post requests was no longer available in middlewares, since version 0.9.0. 

The issue was caused by the removal of the 'body-parser' from the server/defaults.js. After re-adding this parser there, the body became available again in the middleware. I've also added an extra test, to verify that the post body is indeed available in the middleware, to prevent future changes to break this again,

### CODE REVIEW

File: test/cli/index.js [hunk 1]
```diff
@@ -156,6 +157,18 @@ describe('cli', () => {
     })
   })
 
+  describe('db.json -m postbody-middleware.js', () => {
+    beforeEach((done) => {
+      child = cli([ dbFile, '-m', middlewareFiles.postbody ])
+      serverReady(PORT, done)
+    })
+
+    it('Should have post body in middleware', (done) => {
```
Comment (crazy4groovy) [inline test/cli/index.js]: nitpick: lowcase "s" in "should"?

Comment (JHotterbeekx) [inline test/cli/index.js]: I've updated the line, and also updated the code to the new standards. Is it okay to get accepted now? I'd love to have this in the next release.

**Comment by JHotterbeekx** [comment]
Any chance to accept this pull request so we can move on from version 0.8.23?

**Comment by typicode** [comment]
Hi

Sorry for the delay, AFAIR I think it was removed on purpose, because it was conflicting in some cases when the project was used as a module in another Express app (can't remember the issue).

If the idea is to make it available to middlewares loaded by the CLI, would it be possible to make it an option of `jsonServer.defaults`, like `{ bodyParser: true }`, and enable it only in the CLI?

**Comment by JHotterbeekx** [comment]
I've just updated the code that I think you meant for it to be, could you take a look at it?

**Comment by typicode** [comment]
Thank you!

**Comment by andys8** [comment]
@typicode Do you know when these changes will be released?

**Comment by typicode** [comment]
@andys8 Thanks for the reminder, today :)

**Comment by typicode** [comment]
Released :+1:

**Comment by andys8** [comment]
Awesome, @typicode !

**Comment by crazy4groovy** [comment]
So to be clear, I'm using it in this way:

`const middlewares = jsonServer.defaults({ bodyParser: true }); server.use(middlewares);`

**Pull request merged**